Dockerfile

```diff
@@ -4,9 +4,9 @@ RUN apk update && \
     apk upgrade && \
     apk add bash procps drill git coreutils libidn curl socat openssl xxd && \
     rm -rf /var/cache/apk/* && \
-    addgroup testssl && \
-    adduser -G testssl -g "testssl user" -s /bin/bash -D testssl && \
-    ln -s /home/testssl/testssl.sh /usr/local/bin/
+    adduser -D -s /bin/bash testssl && \
+    ln -s /home/testssl/testssl.sh /usr/local/bin/ && \
+    mkdir -m 755 -p /home/testssl/etc /home/testssl/bin
```

Comment:
Just a heads up @drwetter when this PR was raised, I think it was separate from the `mkdir` change, which you had merged earlier.

When you updated the branch here before merging, it looks like it didn't diff well, and accidentally brought back the mkdir line? (_the [previous PR change](https://github.com/drwetter/testssl.sh/pull/2313) is actually [missing from the files history](https://github.com/drwetter/testssl.sh/commits/3.1dev/Dockerfile) now_).

You may want to remove the line again.

Reply:
[Both commits are in the `3.1dev` branch history](https://github.com/drwetter/testssl.sh/commits/3.1dev) though :man_shrugging: